initfiles/componentfiles/configxml/esp.xsl

```diff
@@ -181,7 +183,15 @@
                     </xsl:call-template>
                 </xsl:variable>
                 <xsl:variable name="bindName" select="concat('WSESPControl_Binding_', $process)"/>
+                <xsl:value-of disable-output-escaping="yes" select="$break" />
+                <xsl:value-of disable-output-escaping="yes" select="$indent" />
+                <xsl:value-of disable-output-escaping="yes" select="$indent" />
+                <xsl:value-of disable-output-escaping="yes" select="$indent" />
```

Comment:
the indent is needed if we're going to format the file content explicitly. The fact that eclwatch formats the xml in the viewer makes me wonder if we should bother w/ this change at all...
@RussWhitehead your thoughts please...